docs/tutorial-part2.rst

```diff
@@ -20,7 +21,8 @@ and middleware usage.
 The example application will be a link shortener.
 There will be an option for letting shortened links expire,
 based on time or on the number of times they have been clicked.
-For the sake of simplicity, we'll use the ``shelve`` module
+For the sake of simplicity,
+we'll use the `shelve <https://docs.python.org/3/library/shelve.html>`_ module
```

Comment:
Sphinx has "[intersphinx](https://github.com/mahmoud/clastic/blob/master/docs/conf.py#L44)" links, so this should work: 

```suggestion
we'll use the :mod:`shelve` module
```